### ISSUE TITLE
Thermal image training 

### ISSUE THREAD

**Issue by bushra1100** [matterport/Mask_RCNN#987]
I trained Mask-RCNN over thermal images, but it gives me very bad results with mAP=0.03 and val losses increase continuously.
what should i do to increase mAP?

**Comment by patrickcgray** [comment]
Did you use pretrained model weights? How many training images? Did you use only one channel or multiple? Can you provide some more context?

**Comment by bushra1100** [comment]
yes i did use the pretrained weights, used 146 images for training and 13 for validation. 
i haven't done any changes in channels.  
i am trying to train mask rcnn over thermal images (gray scale), trained the model for 3 epochs , 10 epochs and 30 epochs separately. for 3 epochs it giving mAP=0.03, but for 10 and 20 epochs its not even calculating mAP as some of the validation images went without any detection ( although every image has some persons present). also the loss graphs for validation values continue to increase.

**Comment by patrickcgray** [comment]
I mean image channels as in RGB. The COCO weights are trained on RGB images so it would make sense that with only one channel (grayscale) the CNN would perform very poorly. I would recommend customizing the CNN to only take in tensors with one channel and then you will likely need way more training images because of this change. You might be able to train on some thermal images that don't relate to your specific problem.

What are you trying to detect? Do you have simultaneous thermal and RGB? or Just themal?

**Comment by bushra1100** [comment]
but i have another case in which i have fused RGB and Thermal images which resulted in grayscale. and model is working fine on those fused images.
